AppBidding-MoPub/AppBidding-MoPub/BannerViewController.h

```diff
@@ -0,0 +1,20 @@
+//
+//  BannerViewController.h
+//  AppBidding-MoPub
+//
+//  Copyright © 2020 Criteo. All rights reserved.
+//
+
+
+#ifndef BannerViewController_h
```

Comment:
In all .h files, you're not using such protection against multiple #include. And as you're using #import, that kind of protection is not necessary: https://stackoverflow.com/a/439691.
So I guess that you can remove this